src/src/CustomBuild/define_plugin_sets.h

```diff
@@ -1549,6 +1557,7 @@ To create/register a plugin, you have to :
 #ifdef PLUGIN_SET_COLLECTION_B
     #define USES_P069   // LM75A
 
+    #define USES_P095  // TFT ILI9341, added by TEB 06/28/2024

```

Comment:
Ditto

Reply:
My apologies for including the define_plugin_sets.h file. This was for my personal use only. Please disregard it.